modules/modify-data-object/pkg/utils/parse/clioptions.go

```diff
@@ -42,6 +44,17 @@ func (c *CLIOptions) GetDebugLevel() zapcore.Level {
 	return zapcore.InfoLevel
 }
 
+func (c *CLIOptions) GetScriptTimeout() time.Duration {
+	if c.Timeout != "" {
+		timeout, err := time.ParseDuration(c.Timeout)
+		if err == nil {
+			return timeout
+		}
+	}
+
+	return 0
```

Comment:
@ksimon1 Just to be sure, it should be 0?

Reply:
this case should never happen, but I can put there the default value